l10n_br_account_payment_brcobranca/models/account_journal.py

```diff
@@ -187,11 +188,14 @@ def _get_moves(self, parser, result_row_list):
             for row in result_row:
                 if row.get("type") == "liquidado":
                     data_credito = row.get("date")
-                    break
+                else:
+                    data_credito = row.get("date")
+            if data_credito is None:
+                data_credito = datetime.date.today()
```

Comment:
melhorar esse bloco, dá pra simplificar

Reply:
data_credito = next(
                (
                    row.get("date")
                    for row in result_row
                    if row.get("type") == "liquidado"
                ),
                datetime.date.today(),
            )
Acredito que dessa forma ficaria melhor passando for para descobrir se alguma linha é do tipo liquidação se não vai pegar a data atual